src/lender/coordinator.sol

```diff
@@ -19,38 +19,203 @@ import "tinlake-auth/auth.sol";
 interface EpochTrancheLike {
     function epochUpdate(uint epochID, uint supplyFulfillment_,
         uint redeemFulfillment_, uint tokenPrice_) external;
+    function getTotalOrders(uint epochID) external view returns(uint totalSupply, uint totalRedeem);
 }
 
 interface ReserveLike {
     function updateMaxCurrency(uint currencyAmount) external;
+    function balance() external returns (uint);
```

Comment:
I think from the naming perspective we might should just rename it on the borrower side.

Because `balance` might otherwise be missleading.

`maxBorrowAmount` would be more precise.

wdyt?

Reply:
I changed balance here as well to `totalBalance`